### ISSUE TITLE
Display hostname of Serge server on startup, instead of IP

### ISSUE THREAD

**Issue by cpu123** [serge-web/serge#1650]
## 🐞 Amendment description
On startup Serge displays the IP of the server. This encourages users to navigate to the server via IP, and participants tend to bookmark this during the game. If the IP renews (due to DHCP) this breaks any bookmarks and requires technical support to point out/rectify the issue. Displaying the hostname as well as/instead of the IP would help to prevent this.

## 🔗 Affected page
Serge startup command window

## 🔢 Current behaviour
"I don't know who you are, but I will find you, and I will show you Serge, at 192.168.1.1:8080"

## ⚙️ Expected behavior
"I don't know who you are, but I will find you, and I will show you Serge, at http://[hostname]:8080"

**Comment by IanMayo** [comment]
Note: only the facilitator that starts the Serge executable sees this message.

Good suggestion though, will investigate (though there's a chance we either can't access the hostname, or that there could be more than one hostname assigned).

**Comment by cpu123** [comment]
Yeah, the facilitator has a tendancy to simply pass along the IP that the executable displays to the users - fine for a day or two, but when the IP renews it causes a few headaches! The variable %COMPUTERNAME% should do the trick hopefully?
